projects/spectator/jest/test/matchers/matchers.spec.ts

```diff
@@ -121,6 +122,16 @@ describe('Matchers', () => {
         document.querySelector('custom-element')?.shadowRoot?.querySelector("#shadow-dom")
       ).toBeHidden();
     });
+
+    it('should detect elements whose computed styles are display: none', () => {
+      window.getComputedStyle = () => ({ getPropertyValue: (style) => style == 'display' && 'none' });
```

Comment:
I mocked getComputedStyle here instead of testing an actual <style> element or CSS variable case because `window.getComputedStyle` doesn't appear to work properly in this configuration of JSDOM - it will always return `{}` regardless of element. I did test this method manually in my own project, and it works as expected.